bpf/init.sh

```diff
@@ -93,8 +93,8 @@ ip addr add $HOST_IP/128 dev $HOST_DEV1
 
 ip route del $ADDR/128 dev $HOST_DEV1 2> /dev/null || true
 ip route add $ADDR/128 dev $HOST_DEV1
```

Comment:
@tgraf Sorry I meant this line here, there should be `inet6 fd00::c0a8:210c:0:0/128` in the addresses of the interface.